setup.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2019, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import setuptools
+
+setuptools.setup(
+    name='os2rollekatalog_integration',
+    version='0.1',
+    description='Tool for integrating OS2MO with OS2Rollekatalog',
+    author='Magenta ApS',
+    author_email='info@magenta.dk',
+    packages=setuptools.find_packages(),
+    install_requires=[
+        'os2mo-tools',
```

Comment:
Does this depend on `requests`?

Reply:
However, since this package actually uses `requests`, perhaps it should depend on it directly and not just transitively? What do you think?